GVFS/GVFS.Hooks/HooksPlatform/GVFSHooksPlatform.Mac.cs

```diff
@@ -4,6 +4,11 @@ namespace GVFS.Hooks.HooksPlatform
 {
     public static class GVFSHooksPlatform
     {
+        public static string InstallerExtension()

```

Comment:
Nit: either rename to `GetInstallerExtension` or change to a property with this same name